PODMAN.md

```diff
@@ -23,16 +23,33 @@ satisfactory, consider running podman as root to use plain `overlayfs2`:
 
 ## Rootful podman
 
+In order to use rootful podman by a non root user, we will need to bind podman
+to a socket, accessible by the user (as docker does).
+
+Assuming the user is in `wheel` group please do the following (one time):
+
+As root, create a Drop-In file `/etc/systemd/system/podman.socket.d/10-socketgroup.conf`
+with the following content:
 ```
-mkdir -p $XDG_RUNTIME_DIR/podman
-sudo podman system service -t 0 unix:///$XDG_RUNTIME_DIR/podman/podman.sock
-sudo chown $USER $XDG_RUNTIME_DIR/podman/podman.sock
+[Socket]
+SocketGroup=wheel
+ExecStartPre=/usr/bin/chmod -f 755 /run/podman
```

Comment:
This should be ExecStartPost. Otherwise it may fail on a fresh install due to a chicken egg issue - this systemd file creates the directory, but we are trying to run a command on the (non-existent) directory.

Reply:
Thanks, the `-f` is for this, for some reason it didn't happen here even if I deleted the folder,
anyhow will change as you said because it is a good fix